exercises/G-numbers/exercise.js

```diff
@@ -1 +1,7 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+
+var numberOfStudents =35;
+var numberOfMentors=25;
+
+var total =numberOfStudents + numberOfMentors;
+console.log("Total numnber of students and mentors :"+total)
```

Comment:
Good though it would be better to use the suggested values, 15 and 8 so as to match the expected result.